packages/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/dashboards/addons/ingress-nginx-dashboard.yaml

```diff
@@ -1,12 +1,12 @@
-# Source: 
-{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled .Values.ingressNginx.enabled }}
+{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled .Values.ingressNginx.enabled (or .Values.rkeIngressNginx.enabled .Values.rke2IngressNginx.enabled) }}
```

Comment:
We don't need this right? `.Values.rkeIngressNginx.enabled` can only be set to true if `.Values.ingress.enabled=true`, so we can omit this line's change.